web-console/src/lib/components/streaming/builder/NodeTypes/SqlNode.tsx

```diff
@@ -48,18 +49,18 @@ function SqlTableNode(props: { name: string }) {
       />
       {/* The table- prefix is important for the isValidConnection logic */}
       <Handle
-        id={'table-' + props.name}
+        id={'table-' + escapeRelationName(quotifyRelationName(props.relation))}
         type='target'
         position={Position.Left}
         isConnectable={true}
         isValidConnection={isValidConnection}
-        data-testid={'box-handle-table-' + props.name}
+        data-testid={'box-handle-table-' + quotifyRelationName(props.relation)}
```

Comment:
will this work when relation names contain spaces?

Reply:
Yes :+1: